run.py

```
@@ -277,36 +280,66 @@ def open_browser(ip, port, use_https):
+        print(f"[!] Process scan warning: {e}")
+        
+    # Fallback/Diagnostic: Double check via socket test
+    for p in target_ports:
+        try:
+            s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+            s.bind(('127.0.0.1', p))
+            s.close()
+        except OSError:
+            # Port is still blocked, trigger a system terminal release call for Windows
+            if platform.system() == 'Windows':
```

Ruff (0.16.0)

[error] Undefined name `platform`

(F821)